src/Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php

```diff
@@ -1095,6 +1095,113 @@ public function findServicesByServiceGroups(array $serviceGroups): array
         return $servicesByServiceGroupId;
     }
 
+    /**
+     * @param int $hostId
+     * @param int $serviceId
+     * @return array|null
+     */
+    public function findServiceGroupsByHostAndService(int $hostId, int $serviceId): ?array
+    {
+        $serviceGroups = [];
+
+        if ($this->hasNotEnoughRightsToContinue()) {
+            return $serviceGroups;
+        }
+
+        $sqlExtraParameters = [];
+        $subRequest = '';
+        if (!$this->isAdmin()) {
+            $sqlExtraParameters = [':contact_id' => [\PDO::PARAM_INT => $this->contact->getId()]];
+
+            // Not an admin, we must to filter on contact
+            $subRequest .=
+                ' INNER JOIN `:db`.acl_resources_sg_relations sgr
+                    ON sgr.sg_id = sg.servicegroup_id
+                INNER JOIN `:db`.acl_resources res
+                    ON res.acl_res_id = sgr.acl_res_id
+                    AND res.acl_res_activate = \'1\'
+                    OR res.all_servicegroups = \'1\'
```

Comment:
Not sure it fixed the problem.
Furthermore, with this condition, an acl resource can be handled, even if it's not activate

Reply:
you are right about the activation, I will edit the code and push a fix, thanks